src/cppcache/include/geode/DataOutput.hpp

```diff
@@ -378,9 +378,8 @@ class CPPCACHE_EXPORT DataOutput {
      */
   inline void writeFullUTF(const char* value, uint32_t length = 0) {
```

Comment:
I agree and tried to move it to TcrMessage. However, its implementation depends on private members of DataOutput.